spark/dl/src/main/scala/com/intel/analytics/bigdl/example/lenetLocal/README.md

```diff
@@ -0,0 +1,67 @@
+# Running LeNet5 Model on local JVM
+
+This example shows how to run training, prediction and testing with LeNet5  model on local JVM with BigDL. Lenet5 is a classical CNN model used in digital number classification. For detail information,
+please refer to <http://yann.lecun.com/exdb/lenet/>.
+
+## Prepare MNIST Data
+You can download the MNIST Data from [here](http://yann.lecun.com/exdb/mnist/). Unzip all the
+files and put them in one folder(e.g. mnist).
+
+There're four files. **train-images-idx3-ubyte** contains train images,
+**train-labels-idx1-ubyte** is train label file, **t10k-images-idx3-ubyte** has validation images
+ and **t10k-labels-idx1-ubyte** contains validation labels. For more detail, please refer to the
+ download page.
+
+## Get the JAR
+You can build one by refer to the
+[Build Page](https://github.com/intel-analytics/BigDL/wiki/Build-Page) from the source code.
+
+## Train the Model
+Example command
+```
+java -cp dist/lib/bigdl-VERSION-jar-with-dependencies.jar \
```

Comment:
shall we use bigdl-VERSION-jar-with-dependencies.jar or bigdl-VERSION-jar-with-dependencies-and-spark.jar to for local mode? @yiheng